spec/classes/draggableSpec.js

```diff
@@ -0,0 +1,234 @@
+describe('Draggable', function () {
+    'use strict';
+
+    beforeEach(function () {
+        module('pmImageEditor');
+    });
+
+    describe('factory', function() {
+        var factory,
+            element,
+            event,

```

Comment:
Looks like event is storing coordinates and not an `event`, also event is very generic name, I suggest renaming it to `eventPosition` or something like that more descriptive.